lib/IDE/CodeCompletion.cpp

```diff
@@ -2977,11 +2975,7 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
       return;
 
     if (!D->hasInterfaceType())
-      TypeResolver->resolveDeclSignature(D);
-    else if (isa<TypeAliasDecl>(D)) {
-      // A TypeAliasDecl might have type set, but not the underlying type.
-      TypeResolver->resolveDeclSignature(D);
```

Comment:
What happened to this bit? Just not necessary for other reasons?

Reply:
Yeah, it's been the case for a while that "has interface type" means "has underlying type". Eventually the two will be separate requests.